python/adjoint/objective.py

```diff
@@ -191,17 +205,27 @@ def place_adjoint_source(self, dJ):
         return [source]
 
     def __call__(self):
-        direction = mp.NO_DIRECTION if self.kpoint_func else mp.AUTOMATIC
+        if self.kpoint_func:
+            kpoint_func = self.kpoint_func
+            overlap_idx = self.kpoint_func_overlap_idx
+        else:
+            center_frequency = 0.5 * np.min(self.frequencies) + np.max(
+                self.frequencies)
```

Comment:
Parentheses missing around the sum involving the `np.min` and `np.max` terms?

Reply:
Fixed.